src/Orb/Handler/Handler.hs

```diff
@@ -153,16 +155,17 @@ runPermissionAction handler route body = do
     Right permissionResult -> handleRequest handler route body permissionResult
 
 emptyRequestBodyHandler ::
-  ( HasLogger.HasLogger m
+  ( MIO.MonadIO m
+  , Safe.MonadCatch m
+  , HasLogger.HasLogger m
   , HasRespond.HasRespond m
   , Response.Has500Response tags
-  , UnliftIO.MonadUnliftIO m
   ) =>
   Response.ResponseBodies tags ->
   m (S.TaggedUnion tags) ->
   m Wai.ResponseReceived
 emptyRequestBodyHandler bodies action = do
-  errOrResponse <- UnliftIO.tryAny action
+  errOrResponse <- Safe.tryAny action
```

Comment:
For anyone who might be wondering, this change _should_ have the same behavior around asynchronous exceptions. Which is to say the try here is specialized to handling synchronous exceptions only.